Demo/src/androidTest/java/com/braintreepayments/demo/test/VenmoClientTest.java

```diff
@@ -1,43 +0,0 @@
-package com.braintreepayments.demo.test;
-
-import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
-
-import com.braintreepayments.demo.test.utilities.TestHelper;
-
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static com.braintreepayments.AutomatorAction.click;
-import static com.braintreepayments.AutomatorAssertion.text;
-import static com.braintreepayments.DeviceAutomator.onDevice;
-import static com.braintreepayments.UiObjectMatcher.withResourceId;
-import static com.braintreepayments.UiObjectMatcher.withText;
-import static com.braintreepayments.UiObjectMatcher.withTextStartingWith;
-import static com.braintreepayments.demo.test.utilities.AppInstallationHelper.isAppInstalled;
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.core.StringEndsWith.endsWith;
-import static org.junit.Assume.assumeTrue;
-
-@RunWith(AndroidJUnit4ClassRunner.class)
-public class VenmoClientTest extends TestHelper {
-
-    private static final String VENMO_PACKAGE_NAME = "com.venmo";
-
-    @Ignore("Disable until physical device test automation implemented")
-    @Test(timeout = 120000)
-    public void tokenizesVenmo() {
```

Comment:
We pretty much rely on manual testing for Venmo - this test has been ignored for many years, I think at this point we can remove and revisit in the future if we ever do have device automation